3rdParty/s2geometry/master/src/s2/s2polygon.cc

```diff
@@ -22,6 +22,7 @@
 #include <cmath>
 #include <cstddef>
 #include <memory>
+#include <map>
```

Comment:
I didn't add this to our patches because PR already accepted.
So when we decide to update s2, this issue will be fixed and don't needed patch